controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -490,20 +498,23 @@ func (r *Reconciler) sendToolchainStatusNotification(logger logr.Logger,
 	tsValue := time.Now().Format("20060102150405")
 	contentString := ""
 	subjectString := ""
+	domain := ""
+	if domain, err = removeSchemeFromUrl(toolchainStatus.Status.HostRoutes.ProxyURL); err != nil {
+		return err
```

Comment:
I think in this case we would still want to send the notification even if there was an issue parsing the proxy URL so how about we log the error and set domain to the ClusterURL instead. That way the notification would still be sent to our mailing list.

Reply:
+1 
we could set it to ClusterURL or leave the `domain` empty as it is now

Reply:
Yeah makes sense, instead of returning here, will log the error and set the domain to ClusterURL